include/IECorePython/RefCountedBinding.h

```diff
@@ -63,6 +63,27 @@ class RefCountedWrapper : public T, public WrapperGarbageCollector
 		template<typename Arg1, typename Arg2, typename Arg3>
 		RefCountedWrapper( PyObject *self, Arg1 arg1, Arg2 arg2, Arg3 arg3 );
 
+		template<typename Arg1, typename Arg2, typename Arg3, typename Arg4>
```

Comment:
What would you think of merging as is and then I'll come back and clean it up when I add c++11 (in a couple days ideally)? I'm scrambling to get a bunch of things done this next week before I'm on holiday(ish) for a month, and it seems easier to bash ahead with something that's working.